chains/evm/listener/deposit-handler_test.go

```diff
@@ -224,3 +224,53 @@ func (s *Erc20HandlerTestSuite) TestErc20HandleEventAlternativeAmount() {
 	s.Equal(message, expected)
 	s.Equal(big.NewInt(200), new(big.Int).SetBytes(message.Payload[0].([]byte)))
 }
+
+type Erc1155HandlerTestSuite struct {
+	suite.Suite
+}
+
+func TestRunErc1155HandlerTestSuite(t *testing.T) {
+	suite.Run(t, new(Erc20HandlerTestSuite))
+}
+
+func (s *Erc1155HandlerTestSuite) TestErc1155HandleEvent() {
```

Comment:
Can you please adjust the test names to all be Test_Method_Case (Test_HandleEvent_Case). Also, add what happens if data is malformed please.